include/planes/plane.h

```diff
@@ -207,7 +207,8 @@ void plane_set_pos(struct plane_data* plane, int x, int y);
  * Set the plane scale.
  * You must call plane_apply() to commit the change.
  *
- * @param scale Scale of the plane, where 1.0 is no scale.
+ * @param scale_x Horizontal scale of the plane, where 1.0 is no scale.
+ * @param scale_y Vertical scale of the plane, where 1.0 is no scale.
```

Comment:
Change is fine.  No need to make this a separate commit.  This is all good in one single commit.